aws_lambda_powertools/shared/functions.py

```diff
@@ -1,14 +1,21 @@
 from typing import Any, Optional, Union
 
 
-def strtobool(value):
+def _strtobool(value: str) -> bool:
```

Comment:
please return to `def strtobool` instead of a private one here. This might be useful to others in the future when we have a `commons` utility in v2 or v3